_std/defines/movement.dm

```diff
@@ -0,0 +1,8 @@
+//various sprint flags go here
```

Comment:
stamina.dm could go here

Reply:
i think i want to keep those separate. not everything in stamina belongs in movement, and vice versa

Reply:
let me know if this is a sticking point when i update this!